test/unit-tests/function/algebra/simplify.test.js

```diff
@@ -438,41 +438,78 @@ describe('simplify', function () {
     const optsNCA = { context: { add: { commutative: false } } }
 
-    simplifyAndCompare('x*y+y*x', 'x*y+y*x', {}, optsNCM)
-    simplifyAndCompare('x*y-y*x', 'x*y-y*x', {}, optsNCM)
-    simplifyAndCompare('x*5', 'x*5', {}, optsNCM)
-    simplifyAndCompare('x*y*x^(-1)', 'x*y*x^(-1)', {}, optsNCM)
-    simplifyAndCompare('x*y/x', 'x*y*x^(-1)', {}, optsNCM)
-    simplifyAndCompare('x*y*(1/x)', 'x*y*x^(-1)', {}, optsNCM)
-
-    // Rules apply to *segments* of operands in NC multi-arg. exprs.
-    // ('n*n->n^2')
-    simplifyAndCompare('n*n*3', 'n^2*3', {}, optsNCM)
-    simplifyAndCompare('3*n*n', '3*n^2', {}, optsNCM)
-    simplifyAndCompare('3*n*n*3', '3*n^2*3', {}, optsNCM)
-    // (the following two also subsequently apply 'n^n1*n->n^(n1+1)')
-    simplifyAndCompare('3*n*n*n*3', '3*n^3*3', {}, optsNCM)
-    simplifyAndCompare('3*3*n*n*n*3', '9*n^3*3', {}, optsNCM)
-    simplifyAndCompare('(w*z)*n*n*3', 'w*z*n^2*3', {}, optsNCM)
-    simplifyAndCompare('2*n*n*3*n*n*4', '2*n^2*3*n^2*4', {}, optsNCM) // 'double wedged', +applied >1x
-    // ('v*(v*n1+n2) ->  v^2*n1+v*n2')
-    simplifyAndCompare('w*x*(x*y+z)', 'w*(x^2*y+x*z)', {}, optsNCM)
-    simplifyAndCompare('w*x*(x*y+z)*w', 'w*(x^2*y+x*z)*w', {}, optsNCM)
-    //  'n+n -> 2*n'
-    simplifyAndCompare('x+x+3', '2*x+3', {}, optsNCA)
-    simplifyAndCompare('3+x+x', '3+2*x', {}, optsNCA)
-    simplifyAndCompare('4+x+x+4', '4+2*x+4', {}, optsNCA)
-    simplifyAndCompare('4+x+x+5+x+x+6', '4+2*x+5+2*x+6', {}, optsNCA) // 'double wedged', +applied >1x
-    // 'n+n -> 2*n'  &  'n3*n1 + n3*n2 -> n3*(n1+n2)'
-    simplifyAndCompare('5+x+x+x+x+5', '5+4*x+5', {}, optsNCA)
+    it('should not apply typically applicable rules (for non-commutative contexts)', function () {
+      // i.e. rule 'n3*n1+n3*n2-> n3*(n1+n2)' cannot apply (in this context operands cannot be rearranged)
+      simplifyAndCompare('x*y+y*x', 'x*y+y*x', {}, optsNCM)
+      // i.e. initial temporary/re-arrangement rules 'n-n1->n+-n1','-v->v*-1' do not apply -
+      // resulting in this expr. not being simplified to '0'
+      simplifyAndCompare('x*y-y*x', 'x*y-y*x', {}, optsNCM)
+      // Constants not shifted to left..
+      simplifyAndCompare('x*5', 'x*5', {}, optsNCM)
+      // i.e. 'x*x^-1' not permitted to ultimately cancel-out via 'n*n^n1->n^(n1+1)'
+      simplifyAndCompare('x*y*x^(-1)', 'x*y*x^(-1)', {}, optsNCM)
+      simplifyAndCompare('x*y/x', 'x*y*x^(-1)', {}, optsNCM)
+      simplifyAndCompare('x*y*(1/x)', 'x*y*x^(-1)', {}, optsNCM)
+      // 'n+n->2*n' & 'n1*n3+n2*n3 -> (n1+n2)*n3' cannot apply for NCA
+      simplifyAndCompare('z+2+z', 'z+2+z', {}, optsNCA)
+      simplifyAndCompare('2*z+3+2*z', '2*z+3+2*z', {}, optsNCA)
+    })
 
-    const optsNAA = { context: { add: { associative: false } } }
-    simplifyAndCompare(
-      'x + (-x+y)', 'x + (y-x)', {}, optsNAA, { parenthesis: 'all' })
+    it('should still validly apply (term factoring and collection) rules', function () {
```

Comment:
@josdejong you will have to let me know if this new structure suits you.
This particular section is the only one with added tests; aside from two added in the first block.

Reply:
yes, it makes sense to structure the tests a bit more now that there are more, thanks.